### ISSUE TITLE
the result of WMT en-de  

### ISSUE THREAD

**Issue by travel-go** [facebookresearch/fairseq-lua#116]
Hello,I want to reproduce the result of WMT en-de,but my best bleu is 19.77  when beam is 5.I have done the data bpe operation.
At the same time my training order is
 CUDA_VISIBLE_DEVICES=0 fairseq train -sourcelang en -targetlang de -datadir data-bin/final-one -model fconv -nenclayer 4 -nlayer 3 -dropout 0.2 -optim nag -lr 0.25 -clip 0.1 -momentum 0.99 -timeavg -bptt 0 -maxbatch 1200 -savedir trainings/new-bpe &

This is my result 
| Translated 2737 sentences (74534 tokens) in 101.8s (732.29 tokens/s)
| Timings: setup 0.2s (0.2%), encoder 0.6s (0.6%), decoder 58.6s (57.6%), search_results 0.0s (0.0%), search_prune 32.9s (32.3%)
| BLEU4 = 21.55, 52.3/27.7/16.7/10.3 (BP=0.963, ratio=1.038, sys_len=69201, ref_len=71797)

I do not know what I should do to increase the bleu to 25.16. Should I modify the parameters?

Should I use this?
fairseq train \
   -sourcelang en -targetlang de -datadir ${DATADIR} \
   -model fconv -nembed 768 -nhid 512 \
   -fconv_nhids 512,512,512,512,512,512,512,512,512,1024,1024,1024,1024,2048,2048 \
  -fconv_nlmhids 512,512,512,512,512,512,512,512,512,1024,1024,1024,1024,2048,2048 \
  -fconv_kwidths 3,3,3,3,3,3,3,3,3,3,3,3,3,1,1 \
  -fconv_klmwidths 3,3,3,3,3,3,3,3,3,3,3,3,3,1,1 \
   -optim nag -timeavg -lr 0.25 -momentum 0.99 -clip 0.1 -dropout 0.1 \
  -ngpus 1 -maxbatch 1200 -bptt 0 -batchsize 48 \
  -maxepoch 50 -minepochtoanneal 20 -ndatathreads 4 -seed 1 -log

please help me.Thank you.

**Comment by michaelauli** [comment]
Yes, the model you trained is much smaller than what we used to report results in the paper. You want to use the setting in the README of the pretrained English-German model.

**Comment by travel-go** [comment]
Thank you very much
